### PR TITLE
ci(lint): validate images

### PR DESCRIPTION
**Pull request by jeremylgn**
Restrain possible images format to WEBP, SVG, ICO.
Base verification on the actual format and not on the extension.

### CODE REVIEW

**Review by ad2ien — APPROVED** [review]
😇

**Review by bdeneux — APPROVED** [review]
Nice ! LGTM 👍

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -78,19 +78,18 @@ jobs:
...
-            done
-            if [ "${FAILED}" = "1" ]; then
-              exit 1
+          for file in $(find "$FOLDER" -regextype posix-egrep -regex '.+\.(bmp|ico|jpg|jpeg|png|svg|webp)'); do
+            type=$(identify $file | awk '{print $2}')
+            if [[ $type =~ ^(WEBP|SVG|ICO)$ ]]; then
+              echo "✅ $file"
+            else
+              >&2 echo "❌ $file please use only webp or svg format 🙏"
+              export FAILED=1
```
Comment (ccamel) [inline .github/workflows/lint.yml:87]: No need to export the variable (I know we've done that way for other actions but it's a mistake).

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -78,19 +78,18 @@ jobs:
...
-              fi
-            done
-            if [ "${FAILED}" = "1" ]; then
-              exit 1
+          for file in $(find "$FOLDER" -regextype posix-egrep -regex '.+\.(bmp|ico|jpg|jpeg|png|svg|webp)'); do
+            type=$(identify $file | awk '{print $2}')
+            if [[ $type =~ ^(WEBP|SVG|ICO)$ ]]; then
+              echo "✅ $file"
+            else
+              >&2 echo "❌ $file please use only webp or svg format 🙏"
```
Comment (ccamel) [inline .github/workflows/lint.yml:86]: Just adds a ':' just after the $file to split the sentence.

**Review by ccamel — COMMENTED** [review]
That's great! Thanks for the improvement. 😎

I made a few minor comments, I'll let you have a look.

**Review by ccamel — APPROVED** [review]
:+1:

**Pull request merged**